job/server/src/test/java/alluxio/job/plan/replicate/ReplicateDefinitionTest.java

```diff
@@ -269,32 +269,23 @@ public void runTaskNoBlockWorker() throws Exception {
   }
 
   @Test
-  public void runTaskLocalBlockWorkerPinnedMedium() throws Exception {
-    // file is pinned on a medium
-    mTestStatus.getFileInfo().setMediumTypes(Sets.newHashSet("MEM"));
-    byte[] input = BufferUtils.getIncreasingByteArray(0, (int) TEST_BLOCK_SIZE);
-    TestBlockInStream mockInStream =
-        new TestBlockInStream(input, TEST_BLOCK_ID, input.length, false,
-            BlockInStreamSource.NODE_LOCAL);
-    TestBlockOutStream mockOutStream =
-        new TestBlockOutStream(ByteBuffer.allocate(MAX_BYTES), TEST_BLOCK_SIZE);
-    BlockWorkerInfo localBlockWorker = new BlockWorkerInfo(LOCAL_ADDRESS, TEST_BLOCK_SIZE, 0);
-    runTaskReplicateTestHelper(Lists.newArrayList(localBlockWorker), mockInStream, mockOutStream);
-    assertArrayEquals(input, mockOutStream.getWrittenData());
```

Comment:
Is array compare no longer needed for the test?

Reply:
I added the assert on output array back